manual/manual_circuit.rst

```diff
@@ -1091,7 +1091,7 @@ In practice, it is very common for an experiment to use many circuits with simil
 .. Symbolic parameters of circuits defined as sympy symbols
 .. Gate parameters can use arbitrary symbolic expressions
 
-``sympy`` is a widely-used python package for symbolic expressions and algebraic manipulation, defining :py:class:`sympy.Symbol` objects to represent algebraic variables and using them in :py:class:`sympy.Expression` s to build mathematical statements and arithmetic expressions. Symbolic circuits are managed in ``pytket`` by defining the circuit parameters as :py:class:`sympy.Symbol` s, which can be passed in as arguments to the gates and later substituted for concrete values.
+`sympy <https://docs.sympy.org/latest/index.html>`_ is a widely-used python package for symbolic expressions and algebraic manipulation, defining a sympy :py:class:`~sympy.core.symbol.Symbol` objects to represent algebraic variables and using them in sympy :py:class:`Expression` s to build mathematical statements and arithmetic expressions. Symbolic circuits are managed in ``pytket`` by defining the circuit parameters as :py:class:`sympy.Symbol` s, which can be passed in as arguments to the gates and later substituted for concrete values.
```

Comment:
To be fair, this isn't a very good link because that section of the sympy docs isn't very helpful. Possibly https://docs.sympy.org/latest/explanation/glossary.html#term-Expression is better?

Reply:
I've just used the full link for expression